apps/code-examples/src/app/app.component.html

```diff
@@ -130,6 +130,26 @@
     </ul>
   </li>
 
+  <li>
+    Popovers
+    <ul>
+      <li>
+        Popover
+        <ul>
+          <li>
+            <a routerLink="popovers/popover/asynchronous">Asynchronous</a>
```

Comment:
Should you remove this now that the code example is gone? Or does this point to something else?

Reply:
Yep, good catch.